web/src/views/Overview.vue

```diff
@@ -101,13 +101,13 @@
               </info-dot>
             </h3>
             <div class="stat-card-value">{{ clustering.length }}</div>
-            <div class="stat-card-subtitle text--secondary">Largest cluster: {{ largestCluster }} submissions</div>
+            <div class="stat-card-subtitle text--secondary">Based on current threshold</div>
```

Comment:
This may be too long:
```suggestion
            <div class="stat-card-subtitle text--secondary">File pairs with a similarity above the current threshold are grouped in the same cluster.</div>
```
If so, then the current text is probably good.

Reply:
This is unfortunately too long: 
![image](https://user-images.githubusercontent.com/6013068/187145997-c940bc09-c9dc-40a0-81cd-ff100170d94a.png)